src/components/shared/LinkButton/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
This is pretty much a line for line duplicate of `src/components/shared/Button/index.tsx`, but with `Link`.

Would something like this work instead? We would need to make `onClick` optional for this case.
```
<Link>
  <Button />
</Link>
```

or maybe a dynamic tag that can switch between `Link`, `button`, `a` depending on whether there is an `to`, `onClick`,  `href` prop?

Curious to see @sarboc thoughts

Reply:
I implemented this. Does Button seem too bloated now? That's my only concern.

Reply:
Yeah I hear you. I'm not sure what I like most.

Something I've seen done a lot in OSS is allowing components to have dynamic tags by using React's `createElement` API.

This is idea is pretty similar to the Formik `Field` component. In the `Field` component, we are able to pass our own "presentation" as well as give it any props we want. I think you can also see this pattern in `material-ui`.

Since this button can take 3 different shapes (`a`, `Link`, `button`), it might be an idea worth exploring.

```
React.createElement(
  type,
  [props],
  [...children]
)
```